plugins/action/prettytable.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020, quidame <quidame@poivron.org>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import annotations
+
+from typing import Any, Dict, List, Optional
+
+__metaclass__ = type  # pylint: disable=C0103
```

Comment:
Not sure this is needed in action plugins at the supported versions of ansible-core, but not sure it does not either. @felixfontein ?

Reply:
It is no longer needed.

Reply:
@tgadiev Would you please remove this line? It is no longer needed